### PR TITLE
[onert] Unify FC kernelgen of ACL backends

### PR DESCRIPTION
**Pull request by wateret**
Unify Pool2D kernel generation function to avoid code
duplication.

The unified function is used by 3 pooling op.

- Min
- Max
- L2

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

**Comment by wateret** [comment]
This is for SAM score improvement.

File: runtime/onert/backend/acl_common/AclKernelGen.h [hunk 1]
```diff
@@ -262,6 +263,53 @@ kernelGenFullyConnected(const ir::operation::FullyConnected &node, const ir::Ope
...
+  VERBOSE(Pool2DParam) << "STRIDE_W: " << stride.horizontal << std::endl;
+  VERBOSE(Pool2DParam) << "PAD(T): " << padding.top << std::endl;
+  VERBOSE(Pool2DParam) << "PAD(B): " << padding.bottom << std::endl;
+  VERBOSE(Pool2DParam) << "PAD(L): " << padding.left << std::endl;
+  VERBOSE(Pool2DParam) << "PAD(R): " << padding.right << std::endl;
+
+  auto ofm_tensor = tensor_builder->at(ofm_index).get();
+  auto ifm_tensor = tensor_builder->at(ifm_index).get();
+
+  bool exclude_padding = (pooling_type == ::arm_compute::PoolingType::AVG);
```
Comment (wateret) [inline runtime/onert/backend/acl_common/AclKernelGen.h:300]: I don't know why, but just to keep the same semantics with the original code. (exclude padding is true only for AVG)

Comment (ragmani) [reply runtime/onert/backend/acl_common/AclKernelGen.h:300]: `PoolingLayerInfo`'s `exclude_padding` means output tensor's shape does not have padding. It only have value as its dimension value. 
Our spec of pool operation, however, output tensor's shape have padding. Please always set it as `true`.

Comment (wateret) [reply runtime/onert/backend/acl_common/AclKernelGen.h:300]: Got it, thanks!

**Pull request merged**